src/crypto/algorithms/megolm.js

```diff
@@ -352,12 +363,36 @@ MegolmEncryption.prototype._shareKeyWithDevices = function(session, devicesByUse
             // no devices to send to
             return Promise.resolve();
         }
-
+        function sendToDeviceLoop(slice) {
+            if (slice > maxContentMapId) {
+                return Promise.resolve();
+            }
+            const contentMap = contentMaps[slice];
+            return self._baseApis.sendToDevice("m.room.encrypted", contentMap)
+                .then(() => {
+                    console.log(`Uploaded megolm keys in ${self._roomId} `
+                        + `(slice ${slice}/${maxContentMapId})`);
+
+                    // store that we successfully uploaded the keys of the current slice
+                    for (const userId in contentMap) {
```

Comment:
When I use your suggested line and remove the following `if [...] continue` I get the following error on `npm run lint`:
```
The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype  guard-for-in
```